backend/src/middleware/handleHtmlContent/handleContentData.js

```diff
@@ -47,24 +65,32 @@ const updateHashtagsOfPost = async (postId, hashtags, context) => {
   session.close()
 }
 
-const handleContentData = async (resolve, root, args, context, resolveInfo) => {
-  // extract user ids before xss-middleware removes classes via the following "resolve" call
+const handleContentDataOfPost = async (resolve, root, args, context, resolveInfo) => {
   const idsOfMentionedUsers = extractMentionedUsers(args.content)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-15T22:41:38Z" title="Friday, August 16th 2019, 12:41:38 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -56,15 +63,29 @@ const handleContentData = async (resolve, root, args, context, resolveInfo) => {
   // removes classes from the content
   const post = await resolve(root, args, context, resolveInfo)
 
-  await notify(post.id, idsOfMentionedUsers, context)
+  await notifyMentions('Post', post.id, idsOfMentionedUsers, context)
   await updateHashtagsOfPost(post.id, hashtags, context)
 
   return post
 }
 
+const handleContentDataOfComment = async (resolve, root, args, context, resolveInfo) => {
+  // extract user ids before xss-middleware removes classes via the following "resolve" call
```


I think you can remove those comments after the xss middleware is not removing `.mention` or `.hashtag` anymore.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-16T07:01:26Z" title="Friday, August 16th 2019, 9:01:26 am +02:00">Aug 16, 2019</time>_

---


I removed this comment and similar ones …